sourcetracker/_sourcetracker.py

```diff
@@ -478,22 +501,36 @@ def gibbs_sampler(sink, cp, restarts, draws_per_restart, burnin, delay):
         ordering (same ordering as `final_env_assignments`). The [i, j] entry
         is the environment that the taxon `final_env_assignments[i, j]` is
         determined to have come from in draw i (j is the environment).
-    """
+    times : np.array
```

Comment:
Since you're changing the interface anyway, it might make sense for this to just return a `NamedTuple` object - that's nicer for handling many return values (it's a bit more self-documenting, but since it's still a tuple code that calls this function shouldn't need to be updated).